apps/four_in_a_row.py

```diff
@@ -0,0 +1,205 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2023 Tony Robinson
+
+"""Four in a Row
+~~~~~~~~~~~~~~~~
+ 
+This is the classic two player board game called Four In A Row or Connect4.  You play against the computer.
+
+.. figure:: res/screenshots/FourInARowApp.png
+    :width: 179
+
+    Screenshot of Four In A Row
+
+There is in intro/menu screen which has very brief instructions, allows you to set the opponent level and gives some stats on the number of games you have won.  Touching the screen sets the level from 0 to 6 which corresponds to the number of lookahead plies.  Swiping down enters the main game.  On your turn a red square counter will appear on the top row.  Touch the screen to move to the desired column, optinally touch again if you don't like your choice then swipe down to commit to playing that column.  The computer will reply with a yellow counter after a delay (dependent on level).  Your aim is to get four in a row before the computer does.  At end of game swipe down to return to the intro/menu screen.
```

Comment:
typos
optinally→optionally
plies→piles?

Reply:
Thanks a lot for looking at this.
I've fixed 3 of the 4 suggestions you've made.
The only one I didn't change is 'plies', it has this meaning https://en.wikipedia.org/wiki/Ply_(game_theory)#:~:text=plies.